src/book/02-system/01-foundations/01-overview.mdx

```diff
@@ -10,7 +10,7 @@ The NUbots repository is the main codebase that runs on the robots. Its purpose
 
 The main parts of the NUbots codebase are behaviour, motion, vision, and localisation. Behaviours include strategy, planning and skills. Motion includes kicking, getting up, and walking. Vision includes ball, goal and field detectors. Localisation determines where the robot, ball and the field are in the world through odometry.
 
-At the top level, the robot will use strategy to decide what to do, such as whether it will act as a goalkeeper or if it will act like a striker.
+At the top level, the robot will use strategy to decide what to do, such as whether it will play in the role of goalkeeper, defender or striker.
```

Comment:
Do we have capability to play defender as opposed to striker?

Reply:
Sort of. We have functionality on if we're going offensive or defensive! But not specifically defender/striker. It's more of an example. Maybe I'll try to think of a better example, particularly now that I'm having a look at behaviour.

Reply:
I've changed the sentence.